grakn-core/src/main/java/ai/grakn/engine/TaskId.java

```diff
@@ -49,6 +50,6 @@ public static TaskId generate() {
      * Get the string value of the task ID
      */
     @CheckReturnValue
-    @JsonProperty("value")
+    @JsonValue
```

Comment:
We might need to change `getValue()` into just `value()`, otherwise the JSON representation will have a key `"getValue": "123"`.

Reply:
Really? I thought that `@JsonValue` would serialise the whole object as a String? I will make the change anyway. Not a fan of getters

Reply:
Whoops, you're right!

> ... indicates that results of the annotated "getter" method ... is to be used as the single value to serialize for the instance.

https://fasterxml.github.io/jackson-annotations/javadoc/2.0.0/com/fasterxml/jackson/annotation/JsonValue.html